doc/external/kegg/getKEGGModelForOrganism.html

```diff
@@ -415,1042 +390,953 @@ <h2><a name="_source"></a>SOURCE CODE <a href="#_top"><img alt="^" border="0" sr
+0542     <span class="comment">%can be run on several processors at once</span>
+0543     fastaModel=permuteModel(fastaModel,randperm(RandStream.create(<span class="string">'mrg32k3a'</span>,<span class="string">'Seed'</span>,cputime()),numel(fastaModel.rxns)),<span class="string">'rxns'</span>);
+0544     <a href="constructMultiFasta.html" class="code" title="function constructMultiFasta(model,sourceFile,outputDir)">constructMultiFasta</a>(fastaModel,fullfile(dataDir,<span class="string">'keggdb'</span>,<span class="string">'genes.pep'</span>),fullfile(dataDir,<span class="string">'fasta'</span>));
+0545 <span class="keyword">else</span>
+0546     fprintf(<span class="string">'Generating the KEGG Orthology specific multi-FASTA files... COMPLETE\n'</span>);
+0547 <span class="keyword">end</span>
+0548 
+0549 <span class="keyword">if</span> isunix
+0550     <span class="keyword">if</span> ismac
+0551         binEnd=<span class="string">'.mac'</span>;
+0552     <span class="keyword">else</span>
+0553         binEnd=<span class="string">''</span>;
+0554     <span class="keyword">end</span>
+0555 <span class="keyword">elseif</span> ispc
+0556     binEnd=<span class="string">''</span>;
+0557 <span class="keyword">else</span>
+0558     EM=<span class="string">'Unknown OS, exiting.'</span>;
+0559     disp(EM);
+0560     <span class="keyword">return</span>
+0561 <span class="keyword">end</span>
+0562 
+0563 <span class="comment">%Check if alignment of FASTA files should be performed</span>
+0564 missingAligned=setdiff(KOModel.rxns,[alignedFiles;hmmFiles;alignedWorking;outFiles]);
+0565 <span class="keyword">if</span> ~isempty(missingAligned)
+0566     <span class="keyword">if</span> seqIdentity==-1
+0567         fprintf(<span class="string">'Performing the multiple alignment for KEGG Orthology specific protein sets...   0%% complete'</span>);
+0568     <span class="keyword">else</span>
+0569         fprintf(<span class="string">'Performing clustering and multiple alignment for KEGG Orthology specific protein sets...   0%% complete'</span>);
+0570     <span class="keyword">end</span>
+0571     missingAligned=missingAligned(randperm(RandStream.create(<span class="string">'mrg32k3a'</span>,<span class="string">'Seed'</span>,cputime()),numel(missingAligned)));
+0572     tmpFile=tempname;
+0573     <span class="comment">%On Windows, paths need to be translated to Unix before parsing it to WSL</span>
+0574     <span class="keyword">if</span> ispc
+0575         wslPath.tmpFile=<a href="getWSLpath.html" class="code" title="function path=getWSLpath(path)">getWSLpath</a>(tmpFile);
+0576         <span class="comment">%mafft has problems writing to terminal (/dev/stderr) when running</span>
+0577         <span class="comment">%on WSL via MATLAB, instead write and read progress file</span>
+0578         mafftOutput = tempname;
+0579         wslPath.mafftOutput=<a href="getWSLpath.html" class="code" title="function path=getWSLpath(path)">getWSLpath</a>(mafftOutput));
```

Comment:
```suggestion
0579         wslPath.mafftOutput=<a href="getWSLpath.html" class="code" title="function path=getWSLpath(path)">getWSLpath</a>(mafftOutput);
```

Reply:
Fixed in [fe3daa4](https://github.com/SysBioChalmers/RAVEN/pull/395/commits/fe3daa49f3c6b699b8ff204075c0d6cf04be496a)